packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -0,0 +1,111 @@
+import { useBookerStore } from "../store";
+
+export function Header({ extraDays, isMobile }: { extraDays: number; isMobile: boolean }) {
+  const [layout, setLayout] = useBookerStore((state) => [state.layout, state.setLayout], shallow);
+  const selectedDateString = useBookerStore((state) => state.selectedDate);
+  const addToSelectedDate = useBookerStore((state) => state.addToSelectedDate);
+  const isSmallCalendar = layout === "small_calendar";
+  const selectedDate = dayjs(selectedDateString);
+
+  const onLayoutToggle = useCallback((newLayout: BookerLayout) => setLayout(newLayout), [setLayout]);
+
+  if (isMobile) return null;
+
+  // In month view we only show the layout toggle.
+  if (isSmallCalendar) {
+    return (
+      <div className="fixed top-3 right-3 z-10">
+        <LayoutToggle onLayoutToggle={onLayoutToggle} layout={layout} />
+      </div>
+    );
+  }
+
+  return (
+    <div className="border-subtle relative z-10 flex border-l border-b p-4">
+      <div className="flex items-center gap-3">
+        <h3 className="text-base font-semibold leading-4">
+          {selectedDate.format("MMM D")}-{selectedDate.add(extraDays, "days").format("D")},{" "}
+          <span className="text-subtle">{selectedDate.format("YYYY")}</span>
+        </h3>
+        <ButtonGroup>
+          <Button
+            variant="icon"
+            color="minimal"
+            StartIcon={ChevronLeft}
+            aria-label="Previous Day"
+            onClick={() => addToSelectedDate(-extraDays - 1)}
+          />
+          <Button
+            variant="icon"
+            color="minimal"
+            StartIcon={ChevronRight}
+            aria-label="Next Day"
+            onClick={() => addToSelectedDate(extraDays + 1)}
+          />
+        </ButtonGroup>
+      </div>
+      <div className="ml-auto flex gap-3">
+        <TimeFormatToggle />
+        <div className="fixed top-4 right-4">
+          <LayoutToggle onLayoutToggle={onLayoutToggle} layout={layout} />
+        </div>
+        {/*
+          This second layout toggle is hidden, but needed to reserve the correct spot in the DIV
+          for the fixed toggle above to fit into. If we wouldn't make it fixed in this view, the transition
+          would be really weird, because the element is positioned fixed in the month view, and then
+          when switching layouts wouldn't anymmore, causing it to animate from the center to the top right,
+          while it actuall already was on place. That's why we have this element twice.
+        */}
+        <div className="pointer-events-none opacity-0" aria-hidden>
+          <LayoutToggle onLayoutToggle={onLayoutToggle} layout={layout} />
+        </div>
+      </div>
+    </div>
+  );
+}
+
+const LayoutToggle = ({
+  onLayoutToggle,
+  layout,
+}: {
+  onLayoutToggle: (layout: string) => void;
+  layout: string;
+}) => {
+  const { t } = useLocale();
+
+  return (
+    <div className="[&>div]:bg-muted z-10">
```

Comment:
```suggestion
    <div className="[&>div]:bg-muted z-10">
```

We do this a lot whenever we seem to use ToggleGroup - is it worth just creating a variant that changes the background of the div?

Reply:
I tested this, and actually it isn't needed anymore. `bg-muted` is the default, so the classname didn't do anything anymore here. Removed it.